batch/batch/driver/main.py

```diff
@@ -1152,7 +1152,7 @@ async def cancel_fast_failing_batches(app):
 USER_CORES = pc.Gauge('batch_user_cores', 'Batch user cores', ['state', 'user', 'inst_coll'])
 USER_JOBS = pc.Gauge('batch_user_jobs', 'Batch user jobs', ['state', 'user', 'inst_coll'])
 FREE_CORES = pc.Summary('batch_free_cores', 'Batch instance free cores', ['inst_coll'])
```

Comment:
I'm still having trouble with why we use a  Summary here rather than a Gauge.

Reply:
Hmm. @daniel-goldstein , you're the resident prometheus expert, do you have thoughts? Upon further consideration, it seems to me that this and TOTAL_CORES are both gauges. I don't think I care about the sum over time of the TOTAL_CORES or FREE_CORES (whereas, maybe I do care about that for request latency, like total time users spent waiting over all time?)

Reply:
I'll further note that for this reason `TOTAL_CORES` feels like a not-very-useful Summary, since the further granularity it is showing you over a Gauge is the distribution of machines with different core-counts, which is most likely to always be 100% 16 with maybe a small standing worker. This information is probably best captured with a Gauge that is the number of cores in the collection.

Reply:
Just to be crystal clear here: how does a `Summary` with no labels differ from a `Gauge` with no labels? I think the original code here had hoped that we could use a `Summary` to aggregate over many observations within a small time window. I tried and tried but never figured out how to express that query in prometheus. Maybe I'm missing some key feature? I think our current "batch_utilization" summary *would* be useful if we could somehow say: "collapse all observations that happen at the same time" (but maybe that's note even true because we don't bulk-observe at one time, we observe repeatedly in a loop).